### DISCUSSION TITLE
Charging problem with USB Power Trigger on a GoPro Hero 7 Black

### DISCUSSION THREAD

**Discussion by Vincent44000** [gopro/labs#662] [Q&A]
Hello everyone, I want to automatically start my GoPro 7 black when powered by USB. Using "USB Power Trigger", it works fine, but the camera does not charge anymore! I looked at the charging current with a USB meter and the current remains stuck at 500ma compared to more than 1 amp usually. The problem only seems to appear when I have the USB Power Trigger enabled. Have you ever encountered this problem? Thank you in advance for your answers!

**Comment by dnewman-gpsw** [comment]
Power trigger on HERO7 was not well supported, listed as "Labs enabled HERO7 (limited)" on https://gopro.github.io/labs/control/usb/.  

I don't know how this relates to you power readings, but if the feature is having unwanted effects on your camera, don't use the feature.

**Reply by Vincent44000** [reply]
Yes I read that too... The thing is that for my project, I need a gopro with an integrated HDMI port... I will find another solution. Thank you for your answer

**Reply by Cayrow** [reply]
I also need power to USB with HDMI port output. Im thinking about taking it apart and doing like the FPV drone people do and hardwire a battery eliminator that can also turn it on when power is applied then having a remote shutter button. What did you figure out about this vincent?

**Reply by Cayrow** [reply]
Another solution Im testing right now is a 60 second delay to charge the gopro before starting the footage. I also need the footage to looping. And no wireless signals can be used.

**Reply by Vincent44000** [reply]
Hi Cayrow, I use the USB Power Trigger function... With 500mA the GoPro still charges. Slowly but surely. I just noticed that sometimes the GoPro turns on by itself and stays on until it is completely discharged and does not restart if the USB power comes back...

**Reply by Vincent44000** [reply]
I also thought about dismantling them to tamper with the ignition but I would like to be able to resell them because I have 4 cameras...

**Comment by Cayrow** [comment]
> Hi Cayrow, I use the USB Power Trigger function... With 500mA the GoPro still charges. Slowly but surely. I just noticed that sometimes the GoPro turns on by itself and stays on until it is completely discharged and does not restart if the USB power comes back...

Do you know why they turn on and how often? I don't mind tearing into the GoPros as it is for a project we are selling to a company. I just need the solution to be reliable.

**Reply by Vincent44000** [reply]
For the moment, the problem only seems to appear on one of my Gopros. I noticed this yesterday so I'm doing some testing to find the cause. I'll do cycles for several days and I'll let you know how it went.

**Reply by Vincent44000** [reply]
Before that, I used Sony fdr-x3000r. They have an HDMI port and an API that allows us to turn them on remotely via Bluetooth. It worked very well but the Gopro's HDMI port provided better image quality...

**Reply by Cayrow** [reply]
Yeah. We cant use Bluetooth or Wi-Fi because the company we are selling to restricts wireless connections. Ive seen some guides for FPV and https://youtu.be/V_lejRrYLfc?t=233 this one shows a battery eliminator and it shows where to solder for it to turn on when 5v is applied. Then you might be able to just turn on USB power and start the recording. Not sure if it will work in practice though.

**Reply by Vincent44000** [reply]
This is indeed More reliable solution in your case because on the Gopro 7, if you activate the USB Power Trigger function with the QR code and you turn the camera on or off manually with the button, this deactivates the USB Power Trigger function, which is annoying

**Reply by Cayrow** [reply]
I just talked to my boss and we figured we don't need to record when we ship it out because we can just record off the HDMI using a capture card. That simplifies it as I don't need to record at all on the gopro. We were thinking about using a Keyence camera as we use a lot of Keyence equipment but the stabilization on the camera was just not good enough.
